src/Services/Masa.Auth.Service.Admin/Program.cs

```diff
@@ -144,7 +144,12 @@ await builder.Services
 });
 builder.Services.RemoveAll(typeof(IProcessor));
 
-var app = builder.Services.AddServices(builder);
+var app = builder.AddServices(options => 
+{
+    options.DisableAutoMapRoute = true;
```

Comment:
Do you want to specify it manually? If so, you need add todo comment remind remove it before v1.0.